### DISCUSSION TITLE
mempool-cli things in common

### DISCUSSION THREAD

**Discussion by st3b1t** [st3b1t/SatoshiTop#21] [Show and tell]
I seen this amazing project: https://github.com/mempool/mempool-cli

@qustavo it seems that you have developed in mempool-cli an important part that is missing for now in [SatoshiTop](https://github.com/st3b1t/SatoshiTop) the sequence of blocks..
These are similar projects but written in different languages, it would be interesting to see if there are commonalities at the UX level

**Comment by qustavo** [comment]
I agree. But it seems that integrating a widget written in one language is not trivial. Do you have an idea about how this can be achieved?

**Reply by st3b1t** [reply]
in th emean time... I'm written blocks module, this works in the lasts realeses:
![image](https://github.com/user-attachments/assets/a91fa1c4-24ec-4ec3-a0e4-3773d73d90c2)
the code here:
https://github.com/st3b1t/SatoshiTop/blob/master/lib/bitcoin/blocks.js#L14
